### ISSUE TITLE
[Improvement]:All pages should have a H1 which is not the title of the website.

### ISSUE THREAD

**Issue by ApplePieAngel** [ApplePieAngel/overly-dressed#13]
### Describe whatever you'd like!

All pages should have a H1 which is not the title of the website in order for accessibility best practice.
```
Citation: https://github.com/thedaviddias/Front-End-Checklist
H1: All pages have an H1 which is not the title
```
